.drone.yml

```diff
@@ -99,6 +76,25 @@ steps:
     - success
     - failure
 
+---
```

Comment:
The service definitions need to be in the same pipeline as the build steps that consume them (i.e. above this `---`).
(A pipeline here meaning a discrete list of steps that run on a Drone build -- I know it's an overloaded term)

Reply:
Removed whole section for kube project.